packages/experiments/src/components/FolderCover/FolderCover.scss

```diff
@@ -96,11 +95,13 @@
   &.isSmall {
     font-size: $ms-font-size-s;
     margin-bottom: 3px;
+    @include margin-left(4px);
```

Comment:
All the margins are carefully kept aligned with the code as well. Make sure they're kept up to date!